components/Buttons/PlayButton.vue

```diff
@@ -43,32 +47,57 @@ export default Vue.extend({
       type: Array as () => BaseItemDto[],
       required: true
     },
+    disabled: {
+      type: Boolean,
+      required: false,
+      default: false
+    },
     fab: {
       type: Boolean
     },
     shuffle: {
       type: Boolean
     }
   },
+  data() {
+    return {
+      loading: false
```

Comment:
I think this is a bit superfluous (Hence why it was removed)

When loading a page, the button should simply be disabled (Which is already the case in master, since there's no item to play). There's no need to have two indications (the loading and the disabled state), one is enough and being disabled makes the most sense.

Reply:
No, because this shows the loading circle just on presses. So it doesn't care about the previous stages.

If I enter a library and I'm playing something, the default will still be ``false``. After clicking, PlaybackStatus will be ``Playing``, but that will be skipped as it didn't mutate since the component was initialized. Then, it will go to ``Stopped`` while the queue is being loaded (so, the condition isn't met). Once everything is loaded, playback status will be ``Playing``, so the watcher trigger this time and removes the loading indicator, just like we want.